appengine/parse-server/README.md

```diff
@@ -17,12 +17,12 @@ app to [Google App Engine Managed VMs](https://cloud.google.com/appengine).
 1. `git clone https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git`
 1. `cd appengine/parse-server`
 1. `npm install`
-1. Set the necessary [environment variables](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/appengine/parse-server/server.js#L23).
+1. Set the necessary [environment variables](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/appengine/parse-server/app.yaml#L18).
```

Comment:
We just need them to update config.json once now, both for local and in prod right?